src/auro-hyperlink.js

```diff
@@ -27,6 +27,7 @@ import styleCssFixed from './style-fixed-css.js';
  * @attr {String} href - Specifies the URL of the page link.
  * @attr {String} target - Specifies where to open the linked document.
  * @attr {String} type - Enumerable attribute; [`nav`, `cta`]
+ * @csspart link - apply CSS to the `a` element
```

Comment:
This grammar should fail the linter.

Capitalize the first word and end in a period.
```suggestion
 * @csspart link - Apply CSS to the `a` element.
```